### ISSUE TITLE
Multiplayer Support

### ISSUE THREAD

**Issue by bergsm** [nebriv/VTOLVR-TacviewLogger#30]
This might be a long shot due to the issue stemming from the game detecting the mod loader itself, but it would be nice to get tacview recordings of multiplayer matches. At the moment, with any mods loaded, you can only play in other modded lobbies, of which there are few. To play in the more numerous vanilla lobbies, the mod loader must be disabled.

**Comment by nebriv** [comment]
Yeah unfortunately a limitation in the way the game handles the mods currently. I'd love to see Tacview just implemented natively in the game (or better modding support) :)
